kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/RawCustomResourceOperationsImpl.java

```diff
@@ -745,7 +746,10 @@ private String getLabelsQueryParam(Map<String, String> labels) {
     Request request = (body == null) ? getRequest(url, callMethod) : getRequest(url, body, callMethod);
     try (Response response = client.newCall(request).execute()) {
       if (response.isSuccessful()) {
-        return objectMapper.readValue(response.body().string(), HashMap.class);
+        if(StringUtils.isBlank(response.body().string()))
```

Comment:
Umm, won't `response.body().string()` close the response body?

Reply:
correct, fixing it in few minutes ;)